lib/jsdom/living/webstorage/Storage-impl.js

```diff
@@ -20,6 +20,10 @@ class StorageImpl {
   }
 
   _dispatchStorageEvent(key, oldValue, newValue) {
+    // TODO: clean that up
+    // eslint-disable-next-line
+    const StorageEvent = require("../generated/StorageEvent");
```

Comment:
The new evaluation order introduced by this PR, exhibit the circular dependency between `Storage` and `StorageEvent`. Since this method is not invoked often, it should be fine paying the cost of importing the interface every time the `_dispatchStorageEvent` is invoked.